Packages/com.unity.inputsystem/InputSystem/Editor/InputBindingDrawer.cs

```diff
@@ -9,6 +9,7 @@
 
 ////TODO: reordering support for interactions
 
+#pragma warning disable CS0649
```

Comment:
Preferably put comments on pragmas to make it evident why they are necessary and what they apply to.